--- a/privacyfunctions.cpp
+++ b/privacyfunctions.cpp
@@ -53,7 +53,7 @@                                                                                                    
    // http://freedesktop.org/Standards/Home
    // http://triq.net/~jens/thumbnail-spec/index.html
 
-   QDir thumbnailDir( QDir::homePath() + QLatin1String( "/.thumbnails/normal" ));
+   QDir thumbnailDir( QDir::homePath() + QLatin1String( "/.cache/thumbnails/normal" ));
    thumbnailDir.setFilter( QDir::Files );
    const QStringList entries = thumbnailDir.entryList();
    for( QStringList::const_iterator it = entries.begin() ; it != entries.end() ; ++it) {
@@ -63,7 +63,7 @@
       }
    }
 
-   thumbnailDir.setPath(QDir::homePath() + QLatin1String( "/.thumbnails/large" ));
+   thumbnailDir.setPath(QDir::homePath() + QLatin1String( "/.cache/thumbnails/large" ));
    const QStringList entries2 = thumbnailDir.entryList();
    for( QStringList::const_iterator it = entries2.begin() ; it != entries2.end() ; ++it) {
       if(!thumbnailDir.remove(*it)) {
@@ -72,7 +72,7 @@
       }
    }
 
-   thumbnailDir.setPath(QDir::homePath() + QLatin1String( "/.thumbnails/fail" ));
